Project/src/main/java/app/models/Reviewer.java

```diff
@@ -0,0 +1,30 @@
+package app.models;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class Reviewer extends Role
```

Comment:
Reviewer extending Role as far as I know was meant to be essentially an enum and shouldn't have all of these lists. This info should be in the article under the reviewer, which should point to a User, that way you can just do a findByReviewer in the ArticleRepository.

Reply:
I agree as well

MFW I keep agreeing with Alex 😱